src/loaders/common/loader_helper.py

```diff
@@ -289,6 +290,33 @@ def start_podman_service(uid: int):
     return proc
 
 
+def setup_callback_server_logs():
```

Comment:
This seems really hacky and invasive. Why is it needed when you can stream the logs? https://docker-py.readthedocs.io/en/stable/#getting-started

Reply:
Two different things:

1. setup_callback_server_logs() is for container.conf file. Based on my experiments, we have set it up before we even start podman if we wanna get logs. https://github.com/kbase/collections/blob/5972e2dbbbb6e8412e10940e3dce56f291f53646/src/loaders/workspace_uploader/workspace_uploader.py#L498


2. `stream=True` params is optional here. I converted the entire byte logs to a long string before printing out. `stream=True` will do is have each line into bype, but still need to convert to string. I don't see any huge performance gain from what we currently have.
https://github.com/kbase/collections/blob/5972e2dbbbb6e8412e10940e3dce56f291f53646/src/loaders/workspace_uploader/workspace_uploader.py#L549

Reply:
So by default the containers product no logs?

Reply:
Not that I'm aware of.

Reply:
That's weird. Ok, fine then, but we shouldn't just edit people's conf files - maybe explain what you're about to do and ask for permission first? Or just write out what's needed and stop the script?

Reply:
Sounds reasonable. I will do `ask for permission`.

Reply:
Updated!